tools/integration_tests/log_content/big_file_upload_log_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log_content
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
+)
+
+const (
+	BigFileSize            int64 = 50 * MiB
+	DirForBigFileUploadLog       = "dirForiBigFileUploadLog"
+)
+
+func TestBigFileUploadLog(t *testing.T) {
```

Comment:
what is the pattern we are following @ashmeenkaur ? why do we need seperate files for each test case.
why can't it be upload_log_test.go with 2 test cases?

Reply:
adding why I had to do it.. when I tried to put multiple test functions in the same file, only one of them was running.

Reply:
I see there are multiple tests in other files. Please check why they are not running in your case.

Reply:
@vadlakondaswetha not sure why it wasn't working earlier. It worked this time. So, I've now merged the two tests in one file and renamed it accordingly.